pkg/exchange/paperwallet.go

```diff
@@ -88,18 +90,29 @@ func (p *PaperWallet) Summary() {
 	var (
 		total        float64
 		marketChange float64
+		volume       float64
 	)
 
 	fmt.Println("--------------")
 	fmt.Println("WALLET SUMMARY")
 	fmt.Println("--------------")
+
 	for pair, price := range p.avgPrice {
 		asset, _ := SplitAssetQuote(pair)
 		quantity := p.assets[asset].Free + p.assets[asset].Lock
 		total += quantity * price
 		marketChange += (p.lastCandle[pair].Close - p.fistCandle[pair].Close) / p.fistCandle[pair].Close
 		fmt.Printf("%f %s\n", quantity, asset)
 	}
+
+	fmt.Println()
+	fmt.Println("VOLUMES:")
```

Comment:
Just a small change, it helps users to not confuse with "Volume" from candles.
```suggestion
	fmt.Println("TRADING VOLUME")
```